mediapipe/tasks/ios/core/sources/MPPTaskInfo.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#include "mediapipe/framework/calculator.pb.h"
+#import "mediapipe/tasks/ios/core/sources/MPPTaskOptions.h"
+#import "mediapipe/tasks/ios/core/sources/MPPTaskOptionsProtocol.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Holds all needed informaton to initialize a MediaPipe Task.
+ */
+@interface MPPTaskInfo : NSObject <NSCopying>
+
+@property(nonatomic, copy, nonnull) NSString *taskGraphName;
+
+/**
```

Comment:
This seems to be missing taskName. Is that intentional?

Reply:
[Python implementation](https://github.com/google/mediapipe/blob/master/mediapipe/tasks/python/core/task_info.py#L26) does not have taskName. But Java has. It seems to exist just to designate a name to the task. Doesn't really have an impact on underlying implementation. LMK if it needs to be added.